yt/geometry/_selection_routines/selector_object.pxi

```diff
@@ -1,3 +1,6 @@
+from .oct_visitors cimport CountTotalCells, CountTotalOcts
```

Comment:
Is this because we don't implicitly import from pxd files now?

Reply:
TBH I'm not 100% sure I understand how this file used to compile, but this broke only after I resolved the conflicts from #4195 , so I _assume_ that the dependency graph worked because of misplaced import statements that happened to compensate their absence here.